tests/integration/bigquery.spec.ts

```diff
@@ -87,6 +88,23 @@ describe("@dataform/integration/bigquery", () => {
       ].status
     ).equals(dataform.ActionResult.ExecutionStatus.SUCCESSFUL);
 
+    // Check the status of files expected to execute successfully.
+    expect(
+      actionMap["dataform-integration-tests.df_integration_test.example_incremental"].status
+    ).equals(dataform.ActionResult.ExecutionStatus.SUCCESSFUL);
+    expect(actionMap["dataform-integration-tests.df_integration_test.example_table"].status).equals(
+      dataform.ActionResult.ExecutionStatus.SUCCESSFUL
+    );
+    expect(actionMap["dataform-integration-tests.df_integration_test.example_view"].status).equals(
+      dataform.ActionResult.ExecutionStatus.SUCCESSFUL
+    );
+    expect(actionMap["dataform-integration-tests.df_integration_test.sample_data_2"].status).equals(
+      dataform.ActionResult.ExecutionStatus.SUCCESSFUL
+    );
+    expect(actionMap["dataform-integration-tests.df_integration_test.sample_data"].status).equals(
+      dataform.ActionResult.ExecutionStatus.SUCCESSFUL
+    );
+
```

Comment:
As we discussed, this should have been being tested but weren't, so have added for each adapter